CodeReview/Maru_code_forReview.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
```

Comment:
Fix up to conform to Pep8 style guidelines. Specifically, be careful about tabs vs. spaces.